data/moves.js

```diff
@@ -6818,17 +6818,19 @@ let BattleMovedex = {
 		priority: 0,
 		flags: {},
 		isMax: "Butterfree",
-		onHit(target, source) {
-			for (let pokemon of target.side.active) {
-				let result = this.random(3);
-				if (result === 0) {
-					pokemon.trySetStatus('slp', source);
-				} else if (result === 1) {
-					pokemon.trySetStatus('par', source);
-				} else {
-					pokemon.trySetStatus('psn', source);
+		self: {
+			onHit(source) {
+				for (let pokemon of source.side.foe.active) {
```

Comment:
Is this true? Has anyone tried using this on an ally?

Reply:
Max Moves can't target allies.